pkg/apis/core/types_shoot.go

```diff
@@ -681,6 +681,8 @@ type VerticalPodAutoscaler struct {
 	// The VerticalPodAutoscaler-level maximum allowed takes precedence over the global maximum allowed.
 	// For more information, see https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/docs/examples.md#specifying-global-maximum-allowed-resources-to-prevent-pods-from-being-unschedulable.
 	MaxAllowed corev1.ResourceList
+	// RecommenderUpdateWorkerCount is the number of workers used for updating VPAs and VPACheckpoints in parallel.
```

Comment:
Cosmetic suggestion to unify the doc string of the internal API and the external one.

```suggestion
	// RecommenderUpdateWorkerCount is the number of workers used in the vpa-recommender for updating VPAs and VPACheckpoints in parallel.
```

Reply:
I pushed bd0ca047a29d327ce502b75124bd3e767e08942d to address the cosmetic suggestion.